src/components/calcite-flow/calcite-flow.tsx

```diff
@@ -26,6 +26,34 @@ export class CalciteFlow {
    */
   @Prop({ reflect: true }) theme: CalciteTheme;
 
+  // --------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Removes the currently active `calcite-flow-item`.
+   */
+  @Method()
+  async back(): Promise<HTMLCalciteFlowItemElement> {
+    const lastItem = this.el.querySelector(
+      "calcite-flow-item:last-child"
+    ) as HTMLCalciteFlowItemElement;
+
+    if (!lastItem) {
+      return;
+    }
+
+    const beforeBack = lastItem.beforeBack ? lastItem.beforeBack : () => Promise.resolve();
```

Comment:
Since the flow-item has a default already, is this fallback necessary?

Reply:
Its a nice to have since someone could set the property to null or something.